doc/releases/changelog-0.21.0.md

```diff
@@ -4,22 +4,131 @@
+                                    n_elec, n_qubits)
+  ```
+  ```pycon
+  >>> print(H_tapered)
+    ((-1.7997297644914574+0j)) [I0]
+  + ((-0.10492941956079854+0j)) [X0]
+  + ((0.10492941956079856+0j)) [X1]
+  + ((0.5675134088336165+0j)) [Z1]
+  + ((0.5675134088336168+0j)) [Z0]
+  + ((-0.14563730440190722+0j)) [Y0 Y1]
+  + ((-0.10492941933657857+0j)) [X0 Z1]
+  + ((0.09337410512815508+0j)) [Z0 Z1]
+  + ((0.10492941933657857+0j)) [Z0 X1]
+  >>> hf_tapered
+  tensor([1, 1], requires_grad=True)
+  ```
+
+<h4>New tensor network templates</h4>
+
+* A tensor network templates module has been added. Quantum circuits with the shape
+  of a matrix product state tensor network can now be easily implemented.
+  Motivation and theory can be found in [arXiv:1803.11537](https://arxiv.org/abs/1803.11537).
+  [(#1871)](https://github.com/PennyLaneAI/pennylane/pull/1871)
+
+  An example circuit that uses the `MPS` template is:
+  ```python
+  import pennylane as qml
+  import numpy as np
+
+  def block(weights, wires):
+      qml.CNOT(wires=[wires[0],wires[1]])
+      qml.RY(weights[0], wires=wires[0])
+      qml.RY(weights[1], wires=wires[1])
+
+  n_wires = 4
+  n_block_wires = 2
+  n_params_block = 2
+  template_weights = [[0.1,-0.3],[0.4,0.2],[-0.15,0.5]]
+
+  dev= qml.device('default.qubit',wires=range(n_wires))
+  @qml.qnode(dev)
+  def circuit(weights):
+      qml.MPS(range(n_wires),n_block_wires,block, n_params_block, weights)
+      return qml.expval(qml.PauliZ(wires=n_wires-1))
+  ```
+
+  The resulting circuit is:
+  ```pycon
+  >>> print(qml.draw(circuit,expansion_strategy='device')(template_weights))
+  0: ──╭C──RY(0.1)───────────────────────────────┤
+  1: ──╰X──RY(-0.3)──╭C──RY(0.4)─────────────────┤
+  2: ────────────────╰X──RY(0.2)──╭C──RY(-0.15)──┤
+  3: ─────────────────────────────╰X──RY(0.5)────┤ ⟨Z⟩
+  ```
+
+* Added a template for tree tensor networks (TTN).
+  [(#2043)](https://github.com/PennyLaneAI/pennylane/pull/2043)
+  An example circuit that uses the `TTN` template is:
+  ```python
+  import pennylane as qml
+  import numpy as np
+
+  def block(weights, wires):
+      qml.CNOT(wires=[wires[0],wires[1]])
+      qml.RY(weights[0], wires=wires[0])
+      qml.RY(weights[1], wires=wires[1])
+
+  n_wires = 4
+  n_block_wires = 2
+  n_params_block = 2
+  n_blocks = qml.MPS.get_n_blocks(range(n_wires),n_block_wires)
+  template_weights = [[0.1,-0.3]]*n_blocks
+
+  dev= qml.device('default.qubit',wires=range(n_wires))
+  @qml.qnode(dev)
+  def circuit(template_weights):
+      qml.TTN(range(n_wires), n_block_wires, block, n_params_block, template_weights)
+      return qml.expval(qml.PauliZ(wires=n_wires-1))
+  ```
+  The resulting circuit is:
+  ```pycon
+  >>> print(qml.draw(circuit,expansion_strategy='device')(template_weights))
+  0: ──╭C──RY(0.1)─────────────────┤
+  1: ──╰X──RY(-0.3)──╭C──RY(0.1)───┤
+  2: ──╭C──RY(0.1)───│─────────────┤
+  3: ──╰X──RY(-0.3)──╰X──RY(-0.3)──┤ ⟨Z⟩
+  ```
+
+<h4>More powerful JAX interface</h4>
```

Comment:
How about 'Improved JAX support'? The term 'interface' I think is a bit of jargon for someone casually reading the changelog :)

Reply:
Sg, thanks!